api/src/main/scala/ai/chronon/api/Constants.scala

```diff
@@ -33,6 +33,7 @@ object Constants {
   val TopicInvalidSuffix = "_invalid"
   val lineTab = "\n    "
   val SemanticHashKey = "semantic_hash"
+  val SemanticHashExcludeTopicKey = "semantic_hash_exclude_topic"
```

Comment:
Can we make the boolean flag a map so that we can reuse this for future semantic hash change, like exclude the backfill start date?

Reply:
Great suggestion. Will do